Add Live Tail troubleshooting (#27122)
DataDog/documentation: content/en/logs/troubleshooting/live_tail.md

---
title: Live Tail
description: 'Troubleshoot Logs Live Tail errors or issues'
---

If your Live Tail page shows an error, or the logs do not load, try the following troubleshooting steps:

* Open the Live tail view with your browser in Incognito mode, and see if you can view logs.
* Try a different browser, and check if Live Tail loads logs.
* Check whether other team members can see logs in the Live Tail view.
* Check for any network restrictions, VPN configurations, or antivirus software that may block the Live Tail from loading logs:
	- Ensure that your local or company network allows inbound traffic from `live.logs.datadoghq.com`.
	- Find the Datadog IP prefixes at the [IP ranges][1] endpoint.

## Trace requests

Traceroute is a tool that helps you test the path a packet takes from source to destination. You can use `traceroute` to identify any client-side network issues that may stop Live Tail from loading.

> To examine the route for live tail logs on Linux or MacOS, run the following command:

Reviewer: ```suggestion
To examine the route for Live Tail logs on Linux or MacOS, run the following command:
```
Should Live Tail also be capitalized here in this context ?

{{< code-block lang="shell">}}
traceroute live.logs.datadoghq.com
{{< /code-block >}}

If at any point the request times out, that means the request is blocked somewhere between the client and Datadog servers. Check with your network administration team to address this issue.    

> If a step in the route shows `* * *`, then a host did not reply to the traceroute, or a router does not reply to that protocol. The `* * *` pattern does not always indicate a timeout. Changing the discovery protocol to ICMP/PING using the `-I` option may provide more detailed results.

Reviewer: ```suggestion
If a step in the route shows `* * *`, this could indicate a host did not reply to the traceroute, or a router does not reply to that protocol. The `* * *` pattern does not always indicate a timeout. Changing the discovery protocol to ICMP/PING using the `-I` option may provide more detailed results.
```
Just a suggestion for flow, but your call!

## Clear browser and DNS caches

### Browser cache

Clearing your browser cache may help with Live Tail problems. For example, to clear your cache in Google Chrome, see [Google account help][2].

### DNS cache

Clearing your DNS cache may help with Live Tail problems.

To clear your DNS cache using Google Chrome:
1. Launch the Google Chrome browser.
1. Type `chrome://net-internals/#dns` in the address bar, then click Enter.
1. Click **Clear host cache**.

## Check browser plugins and extensions

Browser plugins and extensions, including ad blockers, sometimes interfere with Live Tail: 
- Check if your browser or operating system has an ad blocker app installed. Uninstall or disable any ad blockers you find, and then check if you are able to see logs in Live Tail.
- Pause, stop, or uninstall any browser plugins and extensions.

## Check access

> Verify that you have a role assigned to you which has the logs [live tail permission][3]. For more information, see [Datadog roles and permissions][4].

Reviewer: ```suggestion
Verify that you have a role assigned to you which has the [logs_live_tail][3] permission. For more information, see [Datadog roles and permissions][4].
```
this is also minor but I wasn't sure it was directing me to the permission called "logs_live_tail" so this could help differentiate that.
